### DISCUSSION TITLE
/sidebar/sanfene/mysql.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#42] [Announcements]
# /sidebar/sanfene/mysql.html

图文详解 66 道 MySQL 面试高频题，这次吊打面试官，我觉得稳了（手动 dog）

https://tobebetterjavaer.com/sidebar/sanfene/mysql.html

**Comment by codepunk-gm** [comment]
65：alter table 表 add column 字段 字段类型 , algorithm=instant; Mysql 8特性 亿级体量加字段毫秒级
